usaspending_api/common/helpers/sql_helpers.py

```diff
@@ -170,68 +166,43 @@ def convert_composable_query_to_string(sql, model=Award, cursor=None):
     return sql
 
 
-def execute_update_sql(sql, model=Award):
-    """
-    Executes a sql query against a database that perform some sort of update
-    (INSERT, UPDATE, etc).
-
-    sql     - Can be either a sql string statement or a psycopg2 Composable
-              object (Identifier, Literal, SQL, etc).
-    model   - A Django model that represents a database table germaine to your
-              query.  If one is not supplied, Award will be used since it is
-              fairly central to the database as a whole.
-
-    Returns a CLOSED cursor.  Can be used for row counts and what not, but is
-    no longer operational.
-    """
-    connection = get_connection(model, False)
-    with connection.cursor() as cursor:
-        # Because django-debug-toolbar does not understand Composable queries,
-        # we need to convert the query to a string before executing it.
-        cursor.execute(convert_composable_query_to_string(sql, cursor=cursor))
-        return cursor
+def cursor_fetcher(cursor):
```

Comment:
What is this for?

Reply:
Originally the update sql convenience function returned a closed cursor instead of the count.  I changed it but left the functionality around.

Reply:
but... the function simply returns the cursor it was provided. What workflow needs this?

Reply:
It's the nature of the beast.  `execute_sql` uses dependency injection to return result.  If, for whatever reason, you want the cursor from `execute_sql` (which I did at one point but later changed my mind), we need a "fetcher" function that can do that.

If it bothers you that much I can remove it and we can add it back in later if  anyone ever needs it.